### PR TITLE
Telia Carrier is now Arelion

### PR DESCRIPTION
**Pull request by jmaslak**

### CODE REVIEW

**Comment by jmaslak** [comment]
I left an old class for Telia there, but am glad to remove it -- not sure what the proper way of handing this rename is, but noticed that the library as-is was not processing Arelion maintenance notices so modified it to do that.

File: README.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ By default, there is a `GenericProvider` that support a `SimpleProcessor` using
 
 #### Supported providers using the BCOP standard
 
+- Arelion
```
Comment (chadell) [inline README.md:61]:
```suggestion
- Arelion (previously Telia)
```

**Review by chadell — APPROVED** [review]
It looks good to me

**Review by scetron — DISMISSED** [review]

**Review by chadell — APPROVED** [review]
LGTM

**Pull request merged**